src/store/MarkerStore.js

```diff
@@ -160,7 +160,8 @@ export default class MarkerStore {
 
         // Y axis positioning logic
         if (this.yPositioner.toLowerCase() === 'none') {
-            this.bottom = undefined;
+            // set the bottom value if there's a bottomWidget, else set it to undefined
+            this.bottom = this.stx.chart.yAxis.initialMarginBottom !== 50 ? 125 : undefined;
```

Comment:
@kellybinary the default margin has been changed to `100` in my last PR #762. So it should be like this
```suggestion
            this.bottom = this.stx.chart.yAxis.initialMarginBottom !== 100 ? 125 : undefined;
```